crawler/constants.py

```diff
@@ -198,3 +227,12 @@
 
 # As per Beckman events detailed in https://ssg-confluence.internal.sanger.ac.uk/display/PSDPUB/Cherrypicking+Events
 PLATE_EVENT_DESTINATION_CREATED: Final[str] = "lh_beckman_cp_destination_created"
+
+
+###
```

Comment:
Was this left by mistake?

Reply:
Not sure what you mean?  These constants are used

Reply:
Oh sorry, because of the empty line on 235 I didn't read the following config as related to the Flask endpoints.